package.json

```diff
@@ -43,7 +44,11 @@
         "react-router-dom": "^4.2.2",
         "react-router-redux": "^4.0.8",
         "redux": "^3.7.2",
-        "redux-thunk": "^2.2.0"
+        "redux-mock-store": "^1.5.1",
+        "redux-thunk": "^2.2.0",
```

Comment:
Refer to this [PR](https://github.com/AndelaOSP/wire/pull/45) on package.json clean up.`moxios` is also a devDependency.

Reply:
Now fixed.